data/core/keymap.lua

```diff
@@ -225,10 +291,19 @@ function keymap.on_key_pressed(k, ...)
         else
           performed = command.perform(cmd, ...)
         end
-        if performed then break end
+        if performed then
+          keymap.current_strokes = nil
+          return true
+        end
+      end
+      -- check if the table contains any further substroke
```

Comment:
I was thinking about something more like `If we're in the middle of a multi-stroke keymap, mark as handled.`